examples/test-app/views/HelloWorld.html

```diff
@@ -1,5 +1,14 @@
-{% from "patterns/header.njk" import header %}
+{% extends "layouts/two_thirds.html" %}
```

Comment:
That change is now published in [@hmcts/look-and-feel v0.2.0](https://github.com/hmcts/look-and-feel/blob/master/CHANGELOG.md#020-2017-08-16)